### DISCUSSION TITLE
Harfbuzz and doom-emacs

### DISCUSSION THREAD

**Discussion by amirmtaati** [harfbuzz/harfbuzz#3604] [Q&A] [answered]
Hi ,
I'm currently using doom-emacs version 3.
I'd like to install and configure harfbuzz in doom-emacs.
I searched the internet a lot but I didn't find anything useful.
Can I get some help please?

**Comment by Eli-Zaretskii** [comment, marked as answer]
Just build and install HarfBuzz, and then rebuild Emacs.

If the above doesn't help, please ask more specific questions (and I'm not sure this is the right place to ask them, unless you have a HarfBuzz-specific issue to ask about).
